mods/fullpotential/scripts.js

```diff
@@ -73,6 +73,9 @@ exports.BattleScripts = {
 		if (move.useTargetOffensive) {
 			attackStat = category === 'Physical' ? 'atk' : 'spa';
 		}
+		if (move.id === 'confusion') {
```

Comment:
Agreed. Wasn't sure what else uses the confusion damage check, so i thought overriding statuses.js would be better.
Will make the change